charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -236,6 +236,7 @@ ztbConfig:
 ztpConfig:
   # Enable akeyless-zero-trust-portal. Valid values: true/false.
   enabled: false
```

Comment:
please add a comment in the line before showing an expected value 
"eu.akeyless.io" .

Reply:
done